nc-games/src/Components/ReviewsByCategory.jsx

```diff
@@ -7,17 +7,21 @@ function ReviewsByCategory({sortBy, setSortBy, orderBy, setOrderBy}) {
     const [items, setItems] = useState([])
     const [isLoading, setIsLoading] = useState(true)
     const {category} = useParams()
+    const [badCategory, setBadCategory] = useState(false)
 
     useEffect(() => {
+        setBadCategory(false)
         setIsLoading(true)
         getReviews(sortBy, orderBy, category).then(items => {
             setItems(items)
             setIsLoading(false);
-        })
+        }).catch(e => {
```

Comment:
you might want to change this slightly so that in the event of a network error it doesn't display your valid categories as invalid